app/src/components/form/Form.js

```diff
@@ -0,0 +1,121 @@
+"use client";
+import Image from "next/image";
+import { useState } from "react";
+import SocialMediaList from "../common/footer/SocialMediaList";
+
+function Form() {
+  const [color, setColor] = useState("#e9dccf");
+  const [label, setLabel] = useState("Enviar");
+
+  const [email, setEmail] = useState("");
+  const [name, setName] = useState("");
+  const [message, setMessage] = useState("");
+
+  function changeStyle() {
+    setColor("#a89582");
+    setLabel("Enviado");
+  }
+
+  return (
+    <div className="quicksand flex justify-center pb-20 pt-20 items-center">
+      <div className="w-1/2">
+        <div className="items-center">
+          <Image
+            alt="Chalet"
+            src="https://images.unsplash.com/photo-1523865236457-3ae3358a4eaa?q=80&w=2070&auto=format&fit=crop&ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D"
+            width={600}
+            height={300}
+            priority={true}
+          />
+          <div className="p-4 text-xs">
+            <p className="pb-2 text-xs">
+              Para mais informações ou dúvidas, entre em contacto: a partir do
+              formulário abaixo; por mensagem privada no Instagram ou envie
+              email para
+            </p>
+            <p>
+              <span className="font-semibold">Phone: </span>913847348
+            </p>
+            <p className="pb-4">
+              <span className="font-semibold">Email: </span>
+              chalet.designinteriores@gmail.com
+            </p>
+            <div>
+              <SocialMediaList />
+            </div>
+          </div>
+        </div>
+      </div>
+
+      <div className="ml-8 w-1/2">
+        <div>
+          <p className="text-2xl opensans pb-10">PECA O SEU ORCAMENTO</p>
+          <form
+            className="w-full"
+            id="freetour"
+            action="https://formcarry.com/s/OlAl-K2DqOh"
```

Comment:
you should use server side props and env vars for this type of information. this should not be public

Reply:
thank you! its done